pkg/3scale/amp/operator/apicast_options_provider.go

```diff
@@ -22,16 +24,23 @@ func NewApicastOptionsProvider(apimanager *appsv1alpha1.APIManager) *ApicastOpti
 }
 
 func (a *ApicastOptionsProvider) GetApicastOptions() (*component.ApicastOptions, error) {
+	imageOpts, err := NewAmpImagesOptionsProvider(a.apimanager).GetAmpImagesOptions()
```

Comment:
This is weird but the alternative would be duplicate some part of the code of GetAmpImageOptions. I guess we can leave it this way to avoid duplication. Let's hope we don't end up having K8s requests in the GetAmpImagesOptions in the long term as we have with other options providers.